lib/web_ui/dev/browser_lock.yaml

```diff
@@ -21,9 +21,19 @@ firefox:
 edge:
   launcher_version: '1.2.0.0'
 ios-safari:
+  # Make sure this version is the same version supported by LUCI macOS bots.
+  # XCode on these bots will be updated once a year, do not forget to update
+  # `heightOfHeader` during this time.
   majorVersion: 13
   minorVersion: 0
   device: 'iPhone 11'
+  # `xcrun simctl` command is used to take screenshots. It takes the screenshot
+  # of the entire simulator. Therefore we need to all the parts other than the
+  # browsers' content. Do not forget to update these values during simulator

```

Comment:
Sure, I think this is still good to write his information here, since on local runs we can also use different phones/tablets to do testing.

@jmagman was working on XCode updates. We can add this information to the documentation.

Reply:
We don't have a doc for updates yet. Feel free to add it in go/flutter-team.

Reply:
Leaving this infor here SGTM, although I'd not phrase it as "do not forget". We can just say "This file must be in sync with the local and LUCI versions of macOS, iOS Simulator, and Xcode. See {LINK_TO_DOCUMENT} for more details about the upgrade process", or something along those lines.

Reply:
I think updating go/flutter-team (and linking it) is a good idea. My only concern is since it's not externally available, is it better to add it to wiki (adding a wiki article such as _"how to update web test harness"_ and  listing all the different steps needed for different browsers (a longer version of [this](https://github.com/flutter/engine/blob/master/lib/web_ui/dev/README.md#upgrade-browser-version)). After fixing the comment, I'll only add a TODO for now.

Reply:
done.